extensions/blocks/subscriptions/index.js

```diff
@@ -29,31 +29,12 @@ export const settings = {
 		<Path d="M23 16v2h-3v3h-2v-3h-3v-2h3v-3h2v3h3zM20 2v9h-4v3h-3v4H4c-1.1 0-2-.9-2-2V2h18zM8 13v-1H4v1h4zm3-3H4v1h7v-1zm0-2H4v1h7V8zm7-4H4v2h14V4z" />
```

Comment:
Pinging @pablohoneyhoney to see if we can take the opportunity to update this icon.